### ISSUE TITLE
Entity Annotations issue

### ISSUE THREAD

**Issue by iyyappanuni** [mradcliffe/agreement#1]
I could see some error on entity annotations, Entity\Agreement.php file. 

below is current code 
```
/**
 * Agreement entity.
 *
 * @ConfigEntityType(
 *   id = "agreement",
 *   label = @Translation("Agreement"),
 *   admin_permission = "administer agreements",
 *   handlers = {
 *     "list_builder" = "Drupal\agreement\Entity\AgreementListBuilder",
 *     "form" => {
 *       "default" = "Drupal\agreement\Entity\AgreementForm",
 *     },
 *     "route_provider" = {
 *       "html" = "Drupal\agreement\Entity\Routing\AgreementRouteProvider",
 *     },
 *   },
 *   config_prefix = "agreement",
 *   static_cache = TRUE,
 *   entity_keys = {
 *     "id" = "name",
 *     "type" = "label",
 *   },
 *   links = {
 *     "delete-form" = "/admin/config/people/agreement/manage/{agreement}/delete",
 *     "edit-form" = "/admin/config/people/agreement/manage/{agreement}",
 *     "collection" = "/admin/config/people/agreement/manage",
 *   },
 *   config_export = {
 *     "name",
 *     "type",
 *     "path",
 *     "settings",
 *     "agreement",
 *   }
 * )
 */
```

**Comment by iyyappanuni** [comment]
Change to 
```

/**
 * Agreement entity.
 *
 * @ConfigEntityType(
 *   id = "agreement",
 *   label = @Translation("Agreement"),
 *   admin_permission = "administer agreements",
 *   handlers = {
 *     "list_builder" = "Drupal\agreement\Entity\AgreementListBuilder",
 *     "form" = {
 *       "default" = "Drupal\agreement\Entity\AgreementForm",
 *     },
 *     "route_provider" = {
 *       "html" = "Drupal\agreement\Entity\Routing\AgreementRouteProvider",
 *     },
 *   },
 *   config_prefix = "agreement",
 *   static_cache = TRUE,
 *   entity_keys = {
 *     "id" = "name",
 *     "type" = "label",
 *   },
 *   links = {
 *     "delete-form" = "/admin/config/people/agreement/manage/{agreement}/delete",
 *     "edit-form" = "/admin/config/people/agreement/manage/{agreement}",
 *     "collection" = "/admin/config/people/agreement/manage",
 *   },
 *   config_export = {
 *     "name",
 *     "type",
 *     "path",
 *     "settings",
 *     "agreement",
 *   }
 * )
 */
```

**Comment by mradcliffe** [comment]
Fixed as part of refactoring in c0b0b9e to make the module actually work. Probably best to uninstall completely if you have it installed somewhere.
